src/main/java/seedu/duke/Duke.java

```diff
@@ -30,18 +28,21 @@ public Duke() {
      */
     public static void main(String[] args) throws IOException {
         new Duke();
+
         favFile.readFile();
-        boolean isOngoing = true;
         try {
             freqFile.readFile();
         } catch (CustomException e) {
             Ui.showError(e);
         }
+
         Ui.printWelcomeMessage();
+        parser = new Parser("lol");
```

Comment:
what is this for?

Reply:
it is just a random initialiser because you need a string within the parser to initialise it

Reply:
I think it'd be good to refactor the lol to final dummy parameter for the sake of readability.